view/vw_qgep_reach.sql

```diff
@@ -1,25 +1,31 @@
-﻿
+
 DROP VIEW IF EXISTS qgep.vw_qgep_reach;
 
 CREATE OR REPLACE VIEW qgep.vw_qgep_reach AS
 
-WITH active_maintenance_event AS (
-  SELECT me.obj_id, me.identifier, me.active_zone, mews.fk_wastewater_structure FROM qgep.od_maintenance_event me
-  LEFT JOIN
-    qgep.re_maintenance_event_wastewater_structure mews ON mews.fk_maintenance_event = me.obj_id
-    WHERE active_zone IS NOT NULL
-)
+--WITH active_maintenance_event AS (
+--  SELECT me.obj_id, me.identifier, me.active_zone, mews.fk_wastewater_structure FROM qgep.od_maintenance_event me
+--  LEFT JOIN
+--    qgep.re_maintenance_event_wastewater_structure mews ON mews.fk_maintenance_event = me.obj_id
+--    WHERE active_zone IS NOT NULL)
```

Comment:
@urskaufmann can you explain why you removed the active_maintenance_event?

Reply:
in the vsa-dss-datamodel or in the is_dictionary_od_field, there is no field active_zone. What is the active_maintenance_event? I think in the normal reach-view, there should not be fields of the me-table. If there is a special usecase, there should be used a special view for this case.

Reply:
This was introduced for @kfischerar for generating maps for maintenance events.
However, this solution (that is being deleted in here) also led to performance issues, so the proposal by @urskaufmann sounds good to me.